Source/Icebreaker/Bot/IcebreakerBot.cs

```diff
@@ -267,16 +284,16 @@ private async Task HandleMessageActivityAsync(ITurnContext turnContext, Cancella
                 else if (string.Equals(activity.Text, MatchingActions.OptIn, StringComparison.InvariantCultureIgnoreCase))
                 {
                     // User opted in
-                    this.telemetryClient.TrackTrace($"User {senderAadId} opted in");
+                    this.telemetryClient.TrackTrace($"User {senderId} opted in");
 
                     var properties = new Dictionary<string, string>
                     {
-                        { "UserAadId", senderAadId },
+                        { "UserAadId", senderId },
```

Comment:
Same as before, please don't label this an AAD Id.

Reply:
No longer making this change